react/components/AvailabilitySubscriber/README.md

```diff
@@ -1,6 +1,7 @@
 # Availability Subscriber
 
 `AvailabilitySubscriber` is a canonical component that any VTEX app can import.
+The submit is made to Master Data on the Entity: `AvailabilitySubscription`
```

Comment:
This is not so simple. You must describe the structure that this component needs on Master Data.